docs/manual/troubleshooting.tex

```diff
@@ -1062,6 +1062,70 @@ \subsection{Code style\label{code-style}}
 in the top-level directory that sets the property \code{maxparallel} to 4000.  If the error is still issued, reduce the number and
 try again.
 
+\section{Upgrading the stubparser module\label{upgrading-stubparser}}
+
+Next steps should be made in order to upgrade the stubparser project:
+\begin{enumerate}
+\item
+Fork \ahref{https://github.com/typetools/stubparser}{the stubparser project} to your github account.
+\item
+Clone the repository to your local environment.
+\begin{Verbatim}
+git clone https://github.com/{user.name}/stubparser
+\end{Verbatim}
+\item
+Enter to the main directory of the local project.
+\begin{Verbatim}
+cd {place-where-repository-cloned}/stubparser
+\end{Verbatim}
+\item
+Create and checkout to the branch (e.g updating).
+\begin{Verbatim}
+git checkout -b updating
+\end{Verbatim}
+\item
+Pull the upstream of \ahref{https://github.com/javaparser/javaparser}{the javaparser project}.
+\begin{Verbatim}
+git pull https://github.com/javaparser/javaparser master
+\end{Verbatim}
+\item
+Resolve the conflicts if required and commit it.
+\item
+Run maven tests in the root stubparser directory.
+\begin{Verbatim}
+mvn test
+\end{Verbatim}
+If something failed in tests the stubparser project should be fixed before continuing.
+\item
+Build the stubparser-core.jar file from the javapaser-core module.
+\item
+Fork \ahref{https://github.com/typetools/checker-framework}{the stubparser project} to your github account.
+\item
+Clone the Checker Framework repository to your local environment.
```

Comment:
It would be better if we could have the Checker Framework as a down-stream dependency for the stubparser project.
Then for each change/PR to the stubparser we'll immediately see whether the CF continues to work.